src/Layers/FeatureLayer/FeatureManager.js

```diff
@@ -235,12 +235,12 @@ export var FeatureManager = VirtualGrid.extend({
 
       pendingRequests--;
 
-      if (pendingRequests <= 0) {
+      if (pendingRequests <= 0 && _visibleZoom()) {
```

Comment:
as you can see in the [failing test](https://travis-ci.org/Esri/esri-leaflet/builds/243430000), `_visibleZoom` isn't defined.

that was a typo on my part. you need to use `this._visibleZoom`.